### ISSUE TITLE
Windows hibernation no longer working after v4.2.4

### ISSUE THREAD

**Issue by jjw24** [utmapp/UTM#5515]
**Describe the issue**  
I was able to use hibernation on Windows 11 Pro with UTM v4.2.4. Subsequent releases it no longer works, every time after the VM is hibernated booting up results in the VM restarting.

I use Windows hibernation as a workaround for not being able to use UTM's Suspend feature due to the use of GPU acceleration. 

**Configuration**  
* UTM Version: v4.3.4
* macOS Version: Ventura 13.1
* Mac Chip (Intel, M1, ...): M1

Happy to provide any additional details if required.

**Comment by osy** [comment]
Did you change the TPM setting?

**Comment by jjw24** [comment]
Hi, I haven't changed the TPM setting. I just did an upgrade from v4.2.4 on the subsequent release versions and started the VM.

**Comment by osy** [comment]
To be clear this is Windows ARM64?

**Comment by jjw24** [comment]
Yes it is.

**Comment by osy** [comment]
By the way, GPU acceleration (the -gl video card) doesn't do much for Windows. If suspend/resume is an important use case, my advice is to stack to the non-gl video card.

**Comment by jjw24** [comment]
Thank you for the tip, I have tried that option before, I am finding without GPU acceleration though the animation is actually very choppy, which makes switching between and using most Windows apps in the VM feel like the experience of using an old computer.

**Comment by osy** [comment]
Hmm, I just tried hibernation on v4.2.5 and it doesn't work either. Can you confirm this?

**Comment by jjw24** [comment]
Yeah I tried all releases after v4.2.4 that are not pre-releases, they all don't work. v4.2.4 is the only version that works. Haven't tried the prior releases though.

**Comment by osy** [comment]
That’s so bizarre. There’s not really any change between 4.2.5 and 4.2.4 except for ANGLE.

**Comment by osy** [comment]
It's not working for me on v4.2.4 either :( I can't fix the issue until I can reproduce it. What is the build number of your Windows?

**Comment by jjw24** [comment]
Of course, happy to provide any information to help you repro the issue. Hoping to get this fixed as I am trying to use UTM
for my dev environment.

I am running Windows version 10.0.22000.2245.

So I found, hibernation may not work on the first go, but does after the second or third try. Additionally, I found leaving the VM booted up for 5 ~ 10 minutes actually helps to get hibernation working. Once hibernation works, it will work consistently.

**Comment by osy** [comment]
> So I found, hibernation may not work on the first go, but does after the second or third try. Additionally, I found leaving the VM booted up for 5 ~ 10 minutes actually helps to get hibernation working. Once hibernation works, it will work consistently.

And this doesn’t work on 4.3.x?

**Comment by jjw24** [comment]
No unfortunately.

**Comment by osy** [comment]
I haven't had time to attempt this yet but can you try v4.3.5? It uses the same QEMU version as v4.2.x

**Comment by jjw24** [comment]
Tried v4.3.5, thank you. Hibernation working as expected. I tested:
1. Bring the VM out of hibernation in v4.3.5 from v4.2.4, no issues
2. Hibernate in v4.3.5 a couple of times and no issues
3. Killed the VM and able to repro the issue I mentioned where if you hibernate again straight after it's booted up then the whole VM restarts. However if waited for 5 minutes roughly, then able to hibernate again and also successful in subsequent tries.
